.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+    -   repo: git@github.com:Yelp/detect-secrets
```

Comment:
to match other repos you can add this https://github.com/uc-cdis/fence/blob/master/.pre-commit-config.yaml#L7-L13
but lgtm